templates/mongodb-atlas.private-endpoint.template.yaml

```
@@ -0,0 +1,267 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation Quickstart with VPC Peering. (qs-1rkorhefe)
+Metadata:
+  cfn-lint: { config: { ignore_checks: [ W9002, W9003, W9006, E3001, E1010 ] } }
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Network Configuration
+        Parameters:
+          - VPCCIDR
+          - PrivateSubnetCIDR
+          - VPCRegion
+      - Label:
+          default: MongoDB Atlas ApiKey Configuration
+        Parameters:
+          - PublicKey
+          - PrivateKey
+          - OrgId
+      - Label:
+          default: MongoDB Atlas Configuration
+        Parameters:
+          - ProjectName
+          - ClusterName
+          - ClusterRegion
+          - ClusterInstanceSize
+          - DatabaseUserRoleDatabaseName
+          - ActivateMongoDBResources
+          - AvailabilityZone
+      - Label:
+          default: AWS Quick Start configuration
+          Parameters:
+            - QSS3BucketName
+            - QSS3KeyPrefix
+            - QSS3BucketRegion
+    ParameterLabels:
+      PublicKey:
+        default: MongoDB Atlas API PublicKey
+      PrivateKey:
+        default: MongoDB Atlas API PrivateKey
+      OrgId:
+        default: MongoDB Atlas API OrgId
+      ProjectName:
+        default: Name of new Atlas Project
+      ClusterName:
+        default: Name of new cluster
+      ClusterRegion:
+        default: The AWS Region for Atlas Cluster
+      ClusterInstanceSize:
+        default: MongoDB Atlas Instance Size
+      DatabaseUserRoleDatabaseName:
+        default: MongoDB Atlas Database User Role Database Name
+      ActivateMongoDBResources:
+        default: Activate MongoDB Atlas CloudFormation resources
+      VPCCIDR:
+        default: VPC CIDR.
+      PrivateSubnetCIDR:
+          default: Private subnet CIDR
+      AvailabilityZone:
+        deafault: AZ fo Subnet
```

Typo and indentation issue.
```suggestion
       default: AZ for subnet
```